phpseclib/File/ANSI.php

```diff
@@ -213,6 +181,16 @@ class File_ANSI
      */
     function File_ANSI()
     {
+        $attr_cell = new stdClass();
+        $attr_cell->bold = false;
+        $attr_cell->underline = false;
+        $attr_cell->blink = false;
+        $attr_cell->background = 'black';
+        $attr_cell->foreground = 'white';
+        $attr_cell->reverse = false;
+        $this->base_attr_cell = clone($attr_cell);
```

Comment:
Is https://github.com/terrafrost/phpseclib/commit/3cb685337a871577cb20fa482efa077645025b3a what you had in mind?